### PR TITLE
[TEMPLATE] Disable ReferenceTopKTest topk_min/max_sort_none on MacOS

### PR DESCRIPTION
**Pull request by sbalandi**
### Details:
 - *temporary disable on MacOS as tests fail in GitHub actions*

### CODE REVIEW

**Comment by ilya-lavrenov** [comment]
is it a regression?

**Comment by sbalandi** [comment]
> is it a regression?

@ilya-lavrenov I can’t say if this was run somewhere before, I don’t see such configuration in nightly, so probably no. I will try to check.

**Comment by ilya-lavrenov** [comment]
These tests are passing on macOS M2 locally.

**Comment by sbalandi** [comment]
@ilya-lavrenov what was your cmake configuration?

I tried on MacOSX 12.6.1 , it was failed for , I build with :
cmake -G "Ninja Multi-Config" -DENABLE_CPPLINT=OFF -DENABLE_NCC_STYLE=OFF -DENABLE_TESTS=ON  -DENABLE_PYTHON=OFF -DCMAKE_VERBOSE_MAKEFILE=ON -DCMAKE_BUILD_TYPE=Release -DBUILD_SHARED_LIBS=ON -DENABLE_ONEDNN_FOR_GPU=OFF  -DCMAKE_COMPILE_WARNING_AS_ERROR=OFF   -DENABLE_STRICT_DEPENDENCIES=OFF   -DENABLE_SYSTEM_SNAPPY=ON  -DENABLE_SYSTEM_TBB=ON -DBUILD_nvidia_plugin=OFF -DENABLE_DEBUG_CAPS=ON  -DOPENVINO_EXTRA_MODULES=../openvino_contrib/modules ..

File: src/plugins/template/tests/functional/skip_tests_config.cpp [hunk 1]
```diff
@@ -131,6 +131,12 @@ std::vector<std::string> disabledTestPatterns() {
     retVector.emplace_back(R"(.*ReferenceTopKTest.*topk_min_sort_none)");
 #endif
 
+#if defined(__APPLE__) || defined(__MACOS)
```
Comment (ilya-lavrenov) [inline src/plugins/template/tests/functional/skip_tests_config.cpp:134]:
```suggestion
#if defined(__APPLE__) && defined(OPENVINO_ARCH_X86_64)
```

**Review by ilya-lavrenov — APPROVED** [review]

File: src/plugins/template/tests/functional/skip_tests_config.cpp [hunk 2]
```diff
@@ -131,6 +131,12 @@ std::vector<std::string> disabledTestPatterns() {
     retVector.emplace_back(R"(.*ReferenceTopKTest.*topk_min_sort_none)");
 #endif
 
+#if defined(__APPLE__) && defined(OPENVINO_ARCH_X86_64)
+    // CVS-64054
```
Comment (ilya-lavrenov) [inline src/plugins/template/tests/functional/skip_tests_config.cpp:135]: this ticket is closed.
should we create another one or reopen existing?

Comment (sbalandi) [reply src/plugins/template/tests/functional/skip_tests_config.cpp]: created new - CVS-120988

**Pull request merged**